src/workflows/pdf/proposal.ts

```diff
@@ -15,25 +15,38 @@ export default function newProposalPdfWorkflowManager(
     userRole
   );
 
+  //Increase the page number by the step.
+  function stepUpToc(toc: TableOfContents[], step: number): TableOfContents[] {
+    if (!toc) {
+      return [];
+    }
+
+    return toc.map((t) => ({
+      ...t,
+      page: t.page != undefined ? t.page + step : undefined, // Todo: Should it be undefined or step?
```

Comment:
Is this still a `TODO` or it is already resolved? It it is resolved please remove the comment if not try to fix it.